### PR TITLE
Remove hidden extract method option.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/2241

This option was set to false, and isn't something the user can change.  Removing it and updating all code (and tests) to correspond go the false value.

### CODE REVIEW

**Comment by mavasani** [comment]
I believe @allisonchou hit this when she was adding extract local function functionality and investigated removal of this unused option - @allisonchou would you be able to review this PR? Thanks!

File: src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs [hunk 1]
```diff
@@ -10800,7 +10849,7 @@ private static IEnumerable<object> NewMethod(ref IEnumerable<object> enumerable)
             // allowMovingDeclaration: false is default behavior on VS.
```
Comment (allisonchou) [inline src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs]: Should the comment here also be removed?

**Comment by CyrusNajmabadi** [comment]
Thanks!

File: src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs]: Yup!  It def should be. :)

**Pull request merged**